go.mod

```diff
@@ -20,7 +20,7 @@ require (
 	github.com/googollee/go-socket.io v1.4.3-0.20191204093753-683f8725b6d0
 	github.com/gorilla/websocket v1.4.1
 	github.com/hashicorp/go-version v1.2.0 // indirect
-	github.com/iotaledger/hive.go v0.0.0-20200118180335-d4e7449f2b6d
+	github.com/iotaledger/hive.go v0.0.0-20200121002816-3019cb9d3e7a
 	github.com/iotaledger/iota.go v1.0.0-beta.13
```

Comment:
`github.com/iotaledger/iota.go v1.0.0-beta.14` got added to `go.sum` but we are using `iota.go v1.0.0-beta.13` here.